build/ballot/1/index.json

```diff
@@ -85,8 +85,9 @@
             "total_loans_received": 9000.0,
```

Comment:
Ignore the built files in this commit.  It was a bad merge.  Fixed in the next commit.